jetty-runner/src/main/java/org/eclipse/jetty/runner/Runner.java

```diff
@@ -164,6 +170,7 @@ public void usage(String error)
         System.err.println(" --out file                          - info/warn/debug log filename (with optional 'yyyy_mm_dd' wildcard");
         System.err.println(" --host name|ip                      - interface to listen on (default is all interfaces)");
         System.err.println(" --port n                            - port to listen on (default 8080)");
+        System.err.println(" --port-file path                    - file to write a single line with the port used to serve http requests ");
```

Comment:
This should probably be more than just the port.
It should likely contain the value of `Server.getURI()` (after `Server.start()`)so that the scheme + network address + port + context path are all communicated.

Reply:
Would `--server-uri-file path` be a better option?